shared/uavobjectdefinition/actuatorsettings.xml

```diff
@@ -23,8 +23,8 @@
             <description>When enabled, the motors will spin at the ChannelNeutral command when armed with zero throttle.</description>
         </field>
 
-        <field name="MotorInputOutputCurveFit" units="-" type="float" elementnames="A,B" defaultvalue="1,1">
-            <description>Actuator mapping of input in [-1,1] to output on [-1,1], using power equation of type a*x^b.</description>
+        <field name="MotorInputOutputCurveFit" units="-" type="float" elements="1" defaultvalue="1">
```

Comment:
<shrug> It's okay here, but in general I think we should all advocate for keeping PRs atomic. From my perspective, this makes it harder for me to track important changes in dRonin that affect what I'm doing. I would have had a surprising settings conflict if I wouldn't have looked into this merge request. As I am this very second trying to track down why a merge conflict broke our internal sensor board build, I'm kinda sensitive to this. :p

I think the justification is to keep things atomically separate. I completely agree that the math works out the same since the scalar factors out. However, the process becomes more complex than it needs to. For instance, the curve fits are better with a second term. So ideally I would still do a two-term curve fit, and then I would neglect the A term in the settings object. Workable, but wonky. 

If it were for some great reason, I'd be all for it. But it's really just to save memory and CPU, right? This is the point where I think it's better to spin off the extremely resource constrained platforms into their own frozen module. I really like https://github.com/TauLabs/TauLabs/issues/1970, but I don't see how that fits into an F1 world.